src/datetime.rs

```diff
@@ -167,10 +167,21 @@ impl crate::DateTime {
         self.0
     }
 
-    pub(crate) fn to_rfc3339(self) -> String {
+    pub fn to_rfc3339_string(self) -> String {
```

Comment:
can we add a docstring to this method? As a general rule we try to document all new public facing types/methods. Right now we don't have an automated lint enabled to verify this because there are a decent number of older things left to be documented, but the eventual plan is to enable one.